pkg/load/load_test.go

```diff
@@ -2,6 +2,7 @@ package load
 
 import (
 	"encoding/json"
+	"github.com/google/go-cmp/cmp"
```

Comment:
:trollface:

Reply:
the imports were not even right in the first place -.-